.eslintrc.json

```diff
@@ -0,0 +1,56 @@
+{
+  "env": {
+      "es6": true,
+      "node": true,
+      "jest": true
```

Comment:
Use this one https://github.com/dojot/dojot/blob/master/storage/time-series/influxdb/retriever/.eslintrc.json